.buildkite/pipeline.yml

```diff
@@ -136,49 +136,37 @@ steps:
 
   # ----------
 
-  # Run CVE checks for the main branch
-  - group: cve image scans
-    steps:
-
-      # scan the operator image for CVEs on merge to main or nightly builds
-      - label: ":buildkite: cve-checks"
-        if: build.branch == "main"
-        depends_on:
-          - "operator-image-build"
-        command: |
-          export CONTAINER=$$(buildkite-agent meta-data get operator-image)
-          cat <<- YAML | buildkite-agent pipeline upload
-          steps:
-            - label: "Trigger cve-slo-status pipeline for $$CONTAINER"
-              trigger: cve-slo-status
-              build:
-                env:
-                  CONTAINER: "$$CONTAINER"
-              soft_fail: true
-          YAML
-        agents:
-          image: docker.elastic.co/ci-agent-images/cloud-k8s-operator/buildkite-agent:f5c5fd8b
-          memory: "256Mi"
-
-      # scan the -ubi operator image on nightly builds
-      - label: ":buildkite: cve-checks"
-        if: build.branch == "main" && build.source == "schedule"
-        depends_on:
-          - "operator-image-build"
-        command: |
-          export CONTAINER=$$(buildkite-agent meta-data get operator-image | sed "s/operator:/operator-ubi:/")
-          cat <<- YAML | buildkite-agent pipeline upload
+  # scan the operator image(s) for CVEs
+  - label: ":buildkite:"
+    if: |  # merge-xyz, tag-bc, tag-final or nightly-main
+      true ||
+      build.branch =~ /^[0-9]*\.[0-9]*\$/
+      || build.tag != null
+      || ( build.branch == "main" && build.source == "schedule")
+    depends_on:
+      - "operator-image-build"
+    command: |
+      list-images() { .buildkite/scripts/build/pre-build-operator.sh | grep -v '#'; }
```

Comment:
> As all the log in build/gen-drivah.toml.sh are prefixed by '#', I added a new log for the image to scan. With that, we can get the images to scan doing: .buildkite/scripts/build/pre-build-operator.sh | grep -v '#'.

Could we store the list of images in a file, and initialize `list-images` with the content of that file?